ShopperInsights/src/main/java/com/braintreepayments/api/ShopperInsightsClient.kt

```diff
@@ -1,6 +1,10 @@
 package com.braintreepayments.api
 
 import androidx.annotation.VisibleForTesting
+import com.braintreepayments.api.ShopperInsightsAnalytics.PAYPAL_PRESENTED
```

Comment:
This may be my lack of Android knowledge but can we just do `import com.braintreepayments.api.ShopperInsightsAnalytics` or do we need to import each event individually?

Reply:
In other modules we have been fully qualifying the events when used (ex: on line 61 `braintreeClient.sendAnalyticsEvent(ShopperInsightsAnalytics.PAYPAL_PRESENTED)`) that being said, either way is fine with me

Reply:
Yep, we could use a wildcard import here `import com.braintreepayments.api.ShopperInsightsAnalytics.*`, but Google is opinionated about import statements. https://developer.android.com/kotlin/style-guide#import_statements

I think there are pros and cons to both ways, but Android Studio is pretty good at formatting imports and auto imports.

Reply:
Nice, thanks for sending over that doc!

Reply:
Some of the cons of using wildcard imports might no longer apply to kotlin, as it gives us a way to use import alias as shown below, but the tools and practices are slower to catch up to the capabilities of the newer languages, so we just go with whatever the tools suggest :D

But yeah, no reason not to use wild card imports in this current scenario.

```
import org.example.Message // Message is accessible
import org.test.Message as TestMessage // TestMessage stands for 'org.test.Message'
```
See: https://kotlinlang.org/docs/packages.html#imports

Reply:
If you are interested, here's how things could go wrong using wildcard imports.
https://www.javadude.com/posts/20040522-import-on-demand-is-evil/

Reply:
There's a way to turn off autogen of `import *` in Android Studio in preferences, I forget exactly where.